frontend/components/puzzle4.jsx

```diff
@@ -13,18 +46,21 @@ class Puzzle4 extends React.Component {
           <p>make the correct selections are you&apos;ll best this foe</p>
         </div>
         <br/>
-        <div className="image-keypad-container">
-          <button id="keypad-1" className="keypad-image">1</button>
-          <button id="keypad-2" className="keypad-image">2</button>
-          <button id="keypad-3" className="keypad-image">3</button>
-          <button id="keypad-4" className="keypad-image">4</button>
-          <button id="keypad-5" className="keypad-image">5</button>
-          <button id="keypad-6" className="keypad-image">6</button>
-          <button id="keypad-7" className="keypad-image">7</button>
-          <button id="keypad-8" className="keypad-image">8</button>
-          <button id="keypad-9" className="keypad-image">9</button>
-          <button id="keypad-0" className="keypad-image">0</button>
-        </div>
+        <KeypadContainer>
+
+          <Button click={this.state.value[0]} onClick={this.handleClick} value="1">1</Button>
+          <Button click={this.state.value[1]} onClick={this.handleClick} value="2">2</Button>
+          <Button click={this.state.value[2]} onClick={this.handleClick} value="3">3</Button>
+          <Button click={this.state.value[3]} onClick={this.handleClick} value="4">4</Button>
+          <Button click={this.state.value[4]} onClick={this.handleClick} value="5">5</Button>
+          <Button click={this.state.value[5]} onClick={this.handleClick} value="6">6</Button>
+          <Button click={this.state.value[6]} onClick={this.handleClick} value="7">7</Button>
+          <Button click={this.state.value[7]} onClick={this.handleClick} value="8">8</Button>
+          <Button click={this.state.value[8]} onClick={this.handleClick} value="9">9</Button>
+
+          <ButtonContainer value={name} onClick={this.handleClick}></ButtonContainer>
```

Comment:
you can ignore the ButtonContainer since this will be delete after I get the disable functionality up and running